tasks/extract_filepaths_for_granule/API.md

```diff
@@ -123,6 +123,7 @@ Lambda handler. Extracts the key's for a granule from an input dict.
   "granules":[
   {
   "granuleId":"granxyz",
+- `"recoverybucketoverride"` - "test-recovery-bucket"
```

Comment:
Extra `` here

Reply:
Looks like it's the wrong format too.

Reply:
I tried building from extract_filepaths_for_granule.py even after fixing there, but does not give the correct format using build_api.sh.
Had to manually format the API.md file